extensions/elytron-security/deployment/src/main/java/io/quarkus/elytron/security/deployment/SecurityDeploymentProcessor.java

```diff
@@ -2,15 +2,13 @@
 
 import java.security.Provider;
 import java.security.Security;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
no widlcard imports

Reply:
Either eclipse-formatter or the maven formatter plugin do this optimization